src/main/java/elon/Elon.java

```diff
@@ -0,0 +1,55 @@
+package elon;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * The main entry point for the Elon application.
+ * Initializes and runs the application, handling user input and interactions.
+ */
+public class Elon {
+    Ui ui = new Ui();
+
+    private Storage storage;
+
+    /**
+     * Constructs an Elon object with the specified Storage.
+     *
+     * @param storage the Storage object used for loading and saving tasks
+     */
+    public Elon(Storage storage) {
+        this.storage = storage;
+    }
+
+
+    /**
+     * Starts the application by greeting the user, loading the task list,
+     * and processing user input commands through the Parser.
+     *
+     * @throws ElonException if an error occurs during command processing
+     */
+    private void Run() throws ElonException {
```

Comment:
Try your best to follow the naming conventions! Methods should start with a lowercase letter and use camelCase.